offer-manager-domain/src/main/groovy/life/qbic/business/offers/OfferContent.groovy

```diff
@@ -0,0 +1,319 @@
+package life.qbic.business.offers
+
+import groovy.transform.EqualsAndHashCode
+import life.qbic.datamodel.dtos.business.AcademicTitle
+import life.qbic.datamodel.dtos.business.Affiliation
+import life.qbic.datamodel.dtos.business.Customer
+import life.qbic.datamodel.dtos.business.ProductItem
+import life.qbic.datamodel.dtos.business.ProjectManager
+
+/**
+ * <h1>A DTO containing the fields required in the offer pdf</h1>
+ *
+ * <p>The content of this class is based on all the fields that need be part in the final offer pdf</p>
+ *
+ * @since 1.1.0
+ *
+*/
+
+@EqualsAndHashCode
+class OfferContent {
+
+    /*Person Information*/
+    /**
+     * The information for the customer for which this offer was created
+     */
+    final String customerFirstName
+    final String customerLastName
+    final String customerTitle
+    /**
+     * The information for the affiliation of the customer selected for this offer
+     */
+    final String customerOrganisation
+    final String customerStreet
+    final String customerPostalCode
+    final String customerCity
+    final String customerCountry
+    /**
+     * The information for the QBiC project manager who was assigned to the project
+     */
+    final String projectManagerFirstName
+    final String projectManagerLastName
+    final String projectManagerTitle
+    final String projectManagerEmail
+
+    /*Project Information*/
+    /**
+     * Date on which the offer was lastly modified
+     */
+    final String creationDate
+    /**
+     * The date on which the offer expires
+     */
+    final String expirationDate
+    /**
+     * The title of the project
+     */
+    final String projectTitle
+    /**
+     * A short objective of the project
+     */
+    final String projectObjective
+    /**
+     * A short description of the experimental design of the project
+     */
+    final String experimentalDesign
+    /**
+     * The identifier for the offer which makes it distinguishable from other offers
+     */
+    final String offerIdentifier
+
+    /*Items*/
+    /**
+     * The items assigned to data generation section of the offer
+     */
+    final List<OfferItem> dataGenerationItems
+    /**
+     * The items assigned to data analysis section of the offer
+     */
+    final List<OfferItem> dataAnalysisItems
+    /**
+     * The items assigned to project management and data storage section of the offer
+     */
+    final List<OfferItem> dataManagementItems
```

Comment:
When you have final lists and you want to prevent modification of the list contents you have to write the getters yourself and cannot delegate that to Groovy.
Please provide getter methods that make sure the list returned cannot be used to modify the dto list

Reply:
done, thanks for pointing this out